docs/api/loaders/ImageBitmapLoader.html

```diff
@@ -29,17 +29,20 @@ <h2>Example</h2>
 		loader.load(
 			// resource URL
 			'textures/skyboxsun25degtest.png',
-			// Function when resource is loaded
+
+			// onLoad callback
 			function ( imageBitmap ) {
 				var texture = new THREE.CanvasTexture( imageBitmap );
 				var material = new THREE.MeshBasicMaterial( { map: texture } );
 			},
-			// Function called when download progresses
+
+			// onProgress callback
```

Comment:
`ImageBitmapLoader` does not support `onprogress` events, too. We should replace this with `undefined`.

Reply:
Good catch!